packages/core-ethereum/crates/core-ethereum-api/Cargo.toml

```diff
@@ -21,8 +21,10 @@ wasm = ["dep:wasm-bindgen", "dep:wasm-bindgen-futures", "dep:gloo-timers",
 prometheus = ["dep:utils-metrics", "core-ethereum-rpc/prometheus"]
 
 [dependencies]
+async-std = { workspace = true }
```

Comment:
When introducing async-std, it'd be future-proof to also introduce a feature that will enable and disable the code, so that it can be tracked in the future and potentially easily swapped out.